loopchain/launcher.py

```diff
@@ -117,18 +118,18 @@ def start_as_rest_server(args):
 
     additional_conf = {
         "port": api_port,
-        "config": conf_path,
         "amqpTarget": conf.AMQP_TARGET,
         "amqpKey": amqp_key,
         "channel": conf.LOOPCHAIN_DEFAULT_CHANNEL,
-        "tbearsMode": False
+        "foreground": True
```

Comment:
not need when call `run_in_background` directly.

Reply:
I see. Thank you.